airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/utils.py

```diff
@@ -302,6 +294,20 @@ def transform_variables(
     return record
 
 
+def transform_col_names(record: Dict, dict_keys: list = []) -> Mapping[str, Any]:
+    """
+    Rename records keys (columns) indicated in `dict_keys` to avoid normalization issues for certain destinations.
+    Example:
+        The `pivot` or `PIVOT` is the reserved keyword for DESTINATION REDSHIFT, we should avoid using it in this case.
+        https://github.com/airbytehq/airbyte/issues/13018
+    """
+    for key in dict_keys:
+        if key in record:
+            record[f"_{key}"] = record[key]  # create new key from original
+            record.pop(key)  # remove the original key
```

Comment:
```suggestion
        record[f"_{key}"] =  record.pop(key) if key in record # create new key from original
```

Reply:
We have to assign the old value first, then remove, this suggestion doesn't cover the assignment part. Leaving as it was.